packages/dev/core/src/Meshes/Compression/dracoDecoder.ts

```diff
@@ -0,0 +1,279 @@
+import { type DecoderModule, DracoDecoderModule } from "draco3dgltf";

```

Comment:
take type out (see my previous suggestion). Same for line 10. It sadly breaks older versions of typescript...